packages/client/vite.config.ts

```diff
@@ -70,11 +70,11 @@ function targetBuildPlugin(dev: boolean, target: string): Plugin {
         const mockPath = `${file.dir}/${file.name}.${target}.${extension}`
         return this.resolve(mockPath, importer)
       } else {
-        const rootPrefix = "src/client/"
+        const sourceRootPrefix = "/src"
```

Comment:
with the new folder structure, this was cutting out import paths like executions.ts